tools/phenomenal/nmr/nmrml2batman.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--Proposed Tool Section: [File Handling]-->
+<tool id="nmrml2batman" name="nmrML2BATMAN Converter">
+     <description>nmrML files to BATMAN converter</description>
+     <requirements>
+       <container type="docker">container-registry.phenomenal-h2020.eu/phnmnl/nmrml2batman</container>
+     </requirements>
+     <command><![CDATA[   
+     export indir=`dirname $nmrMLzip`;
```

Comment:
Didn't notice this... will fix it... shouldn't be creating directories outside of working directory.